packages/cpp/README.md

```diff
@@ -79,6 +79,28 @@ To get this working, you need to enable clangd's global index using the
             "cpp.clangdArgs": "--background-index"
         }
 
+## Using the clang-tidy linter
+
+Note: This functionality is available when using clangd 9 and later.
+
+You can set the preference 'cpp.clang-tidy' to initiate the linter. When the preference is set, there are two possibilities to pass the checks arguments:
+
+- using the preferences:  'cpp.clang-tidy-checks'
+- using the file '.clang-tidy' . The file is located in the same folder of the files or a parent folder.
+
+Note: using the preference checks will supersede the value found in the .clang-tidy file.
+
+The syntax used to fill the checks can be found at http://clang.llvm.org/extra/clang-tidy/
+
+clang-tidy has its own checks and can also run Clang static analyzer checks. Each check has a name and the checks to run can be chosen using the -checks= option, which specifies a comma-separated list of positive and negative (prefixed with -) globs. Positive globs add subsets of checks, negative globs remove them. For example,
```

Comment:
I like the first part. 

But I think we should not mention the `the -checks= option`, since from the PoV of the extension's users, it's a detail they do not need to know (we take care of it for them). Instead we should be telling the users how to use the preferences you added in this PR. 

so a slight rephrasing like: 

"clang-tidy has its own checks and can also run Clang static analyzer checks. Each check has a name (see link above for full list). Clang-tidy takes as input the checks that should run, in the form of  a comma-separated list of positive and negative (prefixed with -) globs. Positive globs add subsets of checks, negative globs remove them. For example:"